terminologies/technique/ultraHighFieldFunctionalMagneticResonanceImaging.jsonld

```diff
@@ -0,0 +1,8 @@
+{
+ "@id":"https://openminds.ebrains.eu/controlledTerms/technique/ultraHighFieldFunctionalMagneticResonanceImaging",

```

Comment:
I would not keep this one. We already have "high field functional magnetic resonance imaging". Unless there is a clear definition between ultra-high-field and high-field.

@tgbugs: what do you think?